Assets/Scripts/CodeBlockMenuManager.cs

```diff
@@ -23,5 +27,9 @@ public class CodeBlockMenuManager : Singleton<CodeBlockMenuManager> {
         public void TurnMenuOn() {
             gameObject.SetActive(true);
```

Comment:
you can just write `SetActive(true);` no need for `gameObject`; similar concept to not having to write `this` in c++

Reply:
<img width="295" alt="setactvie" src="https://user-images.githubusercontent.com/54866570/93010231-05f66500-f53f-11ea-9441-c6d88c2123e9.png">

There's an error when I don't use `gameobject`, so I just left it